### PR TITLE
(patch) fix queries & mutations when using `basePath`

### PR DESCRIPTION
**Pull request by bugzpodder**
Closes: #2048 

### What are the changes and their implications?

Fix rpc client calls for dev/production with basePath
Fix production assets serving with basePath (reverts #2017)

### Checklist

- [x] Changes covered by tests (tests added if needed)
- [ ] PR submitted to [blitzjs.com](https://github.com/blitz-js/blitzjs.com) for any user facing changes

### CODE REVIEW

File: packages/server/src/next-utils.ts [hunk 1]
```diff
@@ -162,7 +162,7 @@ export async function nextStart(nextBin: string, buildFolder: string, config: Se
       rej("")
     } else {
       const nextjs = spawn(nextBin, spawnCommand, {
-        cwd: process.cwd(),
+        cwd: buildFolder,
```
Comment (flybayer) [inline packages/server/src/next-utils.ts:165]: Base path fix looks great!

Was this change needed?

Comment (bugzpodder) [reply packages/server/src/next-utils.ts:165]: @flybayer, yes it seem so.  This will make blitz build + blitz start to serve static assets correctly.  Without this change, blitz start will serve next assets from _next/ instead of ${adminPath}/_next/, while API path is served from ${adminPath}/api and root is served from ${adminPath}/.

Screen Shot 2021-03-08 at 3 45 11 PM
Screen Shot 2021-03-08 at 3 47 38 PM

Comment (flybayer) [reply packages/server/src/next-utils.ts:165]: ok yes, makes sense! Thanks!

**Comment by bugzpodder** [comment]
@all-contributors add @Immortalin for code

**Comment by bugzpodder** [comment]
@all-contributors add @bugzpodder for code
